cypress/support/order-commands.js

```diff
@@ -43,4 +43,5 @@ Cypress.Commands.add("withDeputy", {prevSubject: true}, (order, overrides = {})
           });
       })
   });
+  cy.get("@deputy")
```

Comment:
Just want to check this isn't a typo?

Reply:
No, had to add that bit at the end in order to chain further commands on to `withDeputy` 🤷